examples/device-dashboard/net.c

```diff
@@ -72,6 +72,15 @@ static int event_next(int no, struct event *e) {
   return no + 1;
 }
 
+// This is for newlib and TLS (mbedTLS)
+int _gettimeofday(struct timeval *tv, void *tz) {
+  uint64_t now = mg_millis() + s_boot_timestamp;
+  (void) tz;
+  tv->tv_sec = (time_t) (now / 1000);
+  tv->tv_usec = (unsigned long) ((now % 1000) * 1000);
+  return 0;
+}
+
```

Comment:
This might wreak havoc in other environments, besides, I'm not comfortable with overriding a library call
We've already have problems when overriding time()

Reply:
This is not a library call. This one is with the underscore, it is a newlib "syscall"

Reply:
Yes, of course, we should put it in `syscalls.c`.
I'll propose it later